File: SiriusQuality/SiriusQuality-Priestly-Taylor/src/cpp/SiriusQuality_Priestly_Taylor/PotentialET_PTRate.cpp
#include "PotentialET_PTRate.h"
using namespace SiriusQuality_Priestly_Taylor;


PotentialET_PTRate::PotentialET_PTRate() {}

double PotentialET_PTRate::getevapoTranspirationPriestlyTaylor() { return this->evapoTranspirationPriestlyTaylor; }

void PotentialET_PTRate::setevapoTranspirationPriestlyTaylor(double _evapoTranspirationPriestlyTaylor) { this->evapoTranspirationPriestlyTaylor = _evapoTranspirationPriestlyTaylor; }

File: SiriusQuality/SiriusQuality-Priestly-Taylor/src/java/SiriusQuality_Priestly_Taylor/PotentialET_PTRate.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class PotentialET_PTRate
{
    private double evapoTranspirationPriestlyTaylor;
    
    public PotentialET_PTRate() { }
    
    public PotentialET_PTRate(PotentialET_PTRate toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.evapoTranspirationPriestlyTaylor = toCopy.getevapoTranspirationPriestlyTaylor();
        }
    }
    public double getevapoTranspirationPriestlyTaylor()
    { return evapoTranspirationPriestlyTaylor; }

    public void setevapoTranspirationPriestlyTaylor(double _evapoTranspirationPriestlyTaylor)
    { this.evapoTranspirationPriestlyTaylor= _evapoTranspirationPriestlyTaylor; } 
    
}